ci-operator/step-registry/cucushift/ota/preupgrade/cucushift-ota-preupgrade-commands.sh

```diff
@@ -124,13 +144,8 @@ if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
 fi
 
 set +e
-if [[ "${ENABLE_OTA_TEST}" == "false" ]]; then
-  exit 0
-elif [[ "${ENABLE_OTA_TEST}" == "true" ]]; then
-  extract_oc
+if [[ "${ENABLE_OTA_TEST}" == "true" ]]; then
   run_ota_multi_test
 else
-  extract_oc
```

Comment:
@jiajliu generally we do not need extract oc in pre action, do your case - 66839 need the latest oc version?

Reply:
yes, the case need latest oc for `--included` option

Reply:
@jiajliu if that will move the extract_oc function to OCP-66839 function, is that okay for you?

Reply:
yes, sure. 
btw, what's the expected oc version for other cases in preupgrade step?

Reply:
In my case - OCP-24358, it just get some proxy spec from cluster using oc command, it is a very basic operations, so the built oc command in the verification-test image should be enough. I am not sure the exact version in that image.